### DISCUSSION TITLE
links not working

### DISCUSSION THREAD

**Discussion by jonr1gg1ns** [community/community#57169] [Repositories] [answered]
### Select Topic Area

Question

### Body

i have been following the odin project and i am currently making a website for recipes. However, whenever I try and open any of the recipes from the homepage, i get an error saying "The site configured at this address does not contain the requested file" any ideas on how to fix this?

Website: https://jonr1gg1ns.github.io/odin-recipes/
Code: https://github.com/jonr1gg1ns/odin-recipes

**Comment by airtower-luna** [comment, marked as answer]
The link paths in your `index.html` are absolute (starting with a slash), not relative, so they don't point into the `odin-recipies/` directory, but rather at the top level of your site, where the recipe files don't exist. Remove the leading slashes and they'll work.

See https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls to understand why. :slightly_smiling_face:

**Reply by jonr1gg1ns** [reply]
tyvm this worked
